### ISSUE TITLE
Input validation false positive

### ISSUE THREAD

**Issue by shadyvb** [WordPress-Coding-Standards/WordPress-Coding-Standards#115]
This snippet fails input validation sniff:

``` php
function foo() {
    if ( in_array( $pagenow, $pages ) && ! isset( $_REQUEST['post_type'] ) ) {
        $current_post_type = 'post';
    } else if ( isset( $_REQUEST['post_type'] ) ) {
        $current_post_type = sanitize_key( $_REQUEST['post_type'] );
    }
}
```

Error:

Detected usage of a non-validated input variable: $_REQUEST
